commit 3965b8b71cb6c56f0c41f6f5d96da43da303de03 (refs/changes/90/890/1)
Author: Suk Won Choi <sw00.choi@samsung.com>
Date:   2016-11-14T08:00:54+00:00 (2 years, 11 months ago)
    
    joyent/sdc-docker#95 untagging image used by a container not working

diff --git a/lib/backends/sdc/images.js b/lib/backends/sdc/images.js
index 835cdb2..74c29a7 100644
--- a/lib/backends/sdc/images.js
+++ b/lib/backends/sdc/images.js
@@ -1113,7 +1113,7 @@ function deleteImage(opts, callback) {
                 cb(errors.vmapiErrorWrap(vmapiErr,
                     'could not delete image'));
                 return;
-            } else if (vms.length === 0) {
+            } else if (vms.length === 0 || dontDeleteImages) {
                 cb();
                 return;
             }
